fix(Apple): link to in-app stack trace rules (#9227)
getsentry/sentry-docs: docs/platforms/apple/common/usage/in-app-frames/index.mdx

## Static Frameworks

> When using static frameworks, the frameworks end up in the main executable. Therefore, the SDK currently can't detect if a frame of the main executable

Reviewer: ```suggestion
Because static frameworks end up in the main executable, if you're using one, the Sentry SDK won't be able to detect if a frame of the main executable
```

> originates from your application or a private framework and marks all of them as `inApp`. You can improve your experience by telling Sentry which frames should be marked as `not inApp`, using [Stack Trace Rules](/product/data-management-settings/event-grouping/stack-trace-rules/#stack-trace-rules) on the server.

Reviewer: ```suggestion
originates from your application or a private framework and will mark all frames as `inApp`. To work around this, tell Sentry which frames should be marked as `not inApp`, using [Stack Trace Rules](/product/data-management-settings/event-grouping/stack-trace-rules/#stack-trace-rules) on the server.
```
